test/phpstan-loris.neon

```diff
@@ -0,0 +1,10 @@
+parameters:
+    autoload_directories:
+        - %currentWorkingDirectory%/php/
+        - %currentWorkingDirectory%/src/
+        - %currentWorkingDirectory%/modules/
+        - %currentWorkingDirectory%/test/integrationtests/
+        - %currentWorkingDirectory%/test/unittests/
+    excludes_analyse:
+        - %currentWorkingDirectory%/modules/*/ajax/*
+        - %currentWorkingDirectory%/modules/*/test/*
```

Comment:
I don't see why these are excluded.

Reply:
Just provisionally. The PR is long enough

Reply:
fair enough